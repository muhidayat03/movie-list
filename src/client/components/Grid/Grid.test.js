/* eslint-disable no-undef */
import React from 'react';
import { shallow } from 'enzyme';
import { Col, Row } from './index';




describe('<Row>', () => {
  it('should render correctly ', () => {
    const wrapper = shallow(<Row />);
    expect(wrapper).toMatchSnapshot();
  });
});

describe('<Col>', () => {
  it('should render correctly ', () => {
    const wrapper = shallow(<Col />);
    expect(wrapper).toMatchSnapshot();
  });
});
