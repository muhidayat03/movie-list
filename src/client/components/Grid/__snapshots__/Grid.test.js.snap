// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Col> should render correctly  1`] = `
<div
  className="sc-gsTCUz hCoice"
/>
`;

exports[`<Row> should render correctly  1`] = `
<div
  className="sc-bdfBwQ bMGuAf"
/>
`;
